heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -12,13 +12,29 @@ local msg = {
 Timestamp   = nil,
 Type        = "telemetry",
 Payload     = nil,
-Fields      = { sourceName = "telemetry" }
+EnvVersion  = 1
 }
 
 local UNK_DIM = "UNKNOWN"
 local UNK_GEO = "??"
+local environment_objects = {"build", "settings", "profile", "partner", "system", "addons"}
+local main_ping_objects = {"info", "simpleMeasurements", "histograms", "keyedHistograms", "chromeHangs", "threadHangStats", "log", "fileIOReports", "lateWrites", "addonDetails", "addonHistograms", "UIMeasurements", "slowSQL", "slowSQLstartup", "childPayloads"}
```

Comment:
Nit: split this long list into multiple lines for readabillity, and ideally sort the object names alphabetically (unless the current order has significance).